Makefile

```diff
@@ -41,6 +41,13 @@ metrics:
 integration: ginkgo
 	./ginkgo -v -focus "${FOCUS}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout ${TIMEOUT}
 
+kubernetes:
+	bash .ci/install_bats.sh
+	cd integration/kubernetes && \
```

Comment:
Would be nice to have a script (run_kubernetes.sh) in `.ci` to run this test

Reply:
agree. fixed.